database/src/main/java/entities/Book.java

```diff
@@ -0,0 +1,33 @@
+package entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.io.Serializable;
+
+@Entity
+public class Book implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String title;
+
+    private String authors;
+
+    private String description;
+
+    // add library list (entity), tag list (entity), category id (entity), discount type (enum)
+
+    private Book() {
```

Comment:
In next pull request I will remove this constructor, cause there will be lombok annotation for this purpose.